easy-ui-react/src/Banner/Banner.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { ReactNode } from "react";
+import { Text } from "../Text";
+import { classNames, variationName } from "../utilities/css";
+import styles from "./Banner.module.scss";
+
+export type BannerVariant = "primary" | "success" | "neutral";
```

Comment:
**question:** does `success` make sense in this context?

i know i've been back and forth on the naming of these values in terms of consistency with the rest of the components like the `Button`, but thinking as a consumer rendering a banner as a promotional item, i'm not sure `success` would make sense since it's not the result of an action like other alerts.

it seems like the banners instead are different degrees of prominence. like the green is the most prominent, blue is the next most prominent, followed by the darker blue. which makes me wonder if it should take more of a hierarchical naming path like `<Banner variant="primary" />` (green); `<Banner variant="secondary" />` (blue); `<Banner variant="tertiary" />` (darkblue). or with a little more semantic: `<Banner variant="pivotal" />` (green); `<Banner variant="relevant" />` (blue); `<Banner variant="peripheral" />` (darkblue). those are probably too jargon-y...just brainstorming concepts.

what are your thoughts?

Reply:
good point.

i do see the utility in using `primary` to map to the most used variation, but my concern is that i often feel `primary` refers to a color and it could potentially become confusing. im a bigger fan of the `pivotal` and `relevant` naming, but i went ahead and renamed `variant` to `color` so that maybe its a bit more clear what `primary`, `success` and `neutral` are referring to.  although if you do feel strongly about going to the `variant` naming you suggested with `pivotal` and `relevant`, im on board with that too.

Reply:
i still don't feel super great but i don't have answers lol. might as well just move forward